### ISSUE TITLE
Add new disk stats fields

### ISSUE THREAD

**Issue by SuperQ** [prometheus/procfs#263]
It looks like 5.5 added two new fields to the disk stats.

* https://www.kernel.org/doc/Documentation/ABI/testing/procfs-diskstats
* https://lore.kernel.org/patchwork/patch/1156916/

```
    Kernel 5.5+ appends two more fields for flush requests:

    19 - flush requests completed successfully
    20 - time spent flushing
```

The time is counted in milliseconds.

**Comment by binjip978** [comment]
hello, opened PR regarding this issue #289

**Comment by pgier** [comment]
Resolved by #289
